packages/neos-ui/src/Containers/Modals/SelectNodeType/nodeTypeGroupPanel.js

```diff
@@ -57,9 +58,12 @@ class NodeTypeGroupPanel extends PureComponent {
                 return false;
             });
 
+        // Take `collapsed: true` group seting into account
+        const isOpen = $get('collapsed', group) ? collapsedGroups.includes(name) : !collapsedGroups.includes(name);
```

Comment:
I'm not into this one, but this line looks a bit weird to me:
The panel is *not* collapsed if `group.collapsed` is `true` and it is included in `collapsedGroups` *or* if `group.collapsed` is _not_ `true` and it is _not_ included in `collapsedGroups`.
Is that correct and does it make sense?

Reply:
Hey @bwaidelich, sorry for writing unclear code, was just lazy to do the proper refactoring.
The collapsedGroups in this case become more like toggledGroups: it takes the default value specified by the setting, and flips it. So it's basically an XNOR:
`isOpen = $get('collapsed', group) XNOR toggledGroups.includes(name)`

I've created a PR to make the code a bit more readable: https://github.com/neos/neos-ui/pull/2070